js/File.js

```diff
@@ -2,11 +2,12 @@ class File {
     // Canvas, canvas state
     canvasSize = [];
     zoom = 7;
-    canvasView = document.getElementById("canvas-view");
+    canvasView = document.getElementById("canvas-view") ?? document.createElement("canvas");
```

Comment:
canvasView is a div, not a canvas, but other than that I'd avoid error checking on DOM elements, creating an empty element only delays the inevitable in my opinion